packages/js/components/src/sortable/sortable.tsx

```diff
@@ -51,7 +51,7 @@ export const Sortable = ( {
 }: SortableProps ) => {
 	const ref = useRef< HTMLOListElement >( null );
 	const [ items, setItems ] = useState< SortableChild[] >( [] );
-	const [ selectedIndex, setSelectedIndex ] = useState< number >( 0 );
+	const [ selectedIndex, setSelectedIndex ] = useState< number >( -1 );
```

Comment:
@joelclimbsthings @joshuatf I set the default to `-1` instead of `0` as this would cause it to be highlighted right away, which didn't quite make sense to me.
But let me know if I should change this?